policy/leaving-governance.md

```diff
@@ -4,13 +4,24 @@ This documents the process of leaving Electron Governance. A contributor is no l
 
 ## Offboarding Items
 
-* Update README file for the Working Group. Typically the current WG chair will submit this, but any member of the WG can submit.
+1. A governance member (Typically the WG Chair) update README file in [governance](https://github.com/electron/governance) for the WG the member is leaving.
```

Comment:
- 'Typically' is capitalized for no reason
- grammar error with "member update README file"
- doesn't discuss what change is actually to be made in the README. Does the member get removed? Do we have a separate section for former members? There was discussion in Vancouver about emeritus members; how are those handled?